src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx

```diff
@@ -57,8 +57,32 @@ binary](https://github.com/newrelic/newrelic-snowflake-integration/releases). Pl
     title="Step 3. Update `config.yaml`"
     >
 To ensure you're using the most recent information for the integration, update your `config.yaml` file so that the file matches the snippet found in Snowflake's [Key Pair authentication](https://github.com/newrelic/newrelic-snowflake-integration#key-pair-authentication) section. 
+    </Collapser>
+    <Collapser
+    id="step4"
+    title="Step 4. When the agent is running as a systemd service"
+    >      
+If running the New Relic infrastructure agent as a systemd service, follow these [steps to set the environment variable](https://github.com/newrelic/newrelic-snowflake-integration#when-the-agent-is-running-as-a-systemd-service)      
+i.To determine if you are on a system using `systemd` as the init service, run 
```

Comment:
May I know, what does "i." refer to ?
As there are no succeeding points after this, I feel it could be removed.
@rahulbasu-nr , requesting your expertise on this.

Thank you 👍